### PR TITLE
[SupportBundle] Sets email address of contact in replyTo field of sent mails

### PR DESCRIPTION
**Pull request by kitan1982**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Fixed tickets | #1194 

CoreBundle :
- Allows to add a email address in the "replyTo" options in the "send mail" function

SupportBundle :
- Sets contact email address defined in ticket as the "replyTo" address of the notification mails
- Adds a placeholder for description field in ticket form

### CODE REVIEW

File: plugin/support/Controller/AdminSupportController.php [hunk 1]
```diff
@@ -86,6 +86,7 @@ public function __construct(
      */
     public function adminSupportIndexAction()
     {
+        throw new \Exception('mdr');
```
Comment (stefk) [inline plugin/support/Controller/AdminSupportController.php]: !

Comment (kitan1982) [inline plugin/support/Controller/AdminSupportController.php]: Done

**Comment by LaurentGruber** [comment]
LGTM

**Comment by ngodfraind** [comment]
LGTM

**Pull request merged**